extensions/amp-truncate-text/1.0/amp-truncate-text.css

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+amp-truncate-text {
+  padding: 0 !important;
+  border: none !important;
+}
+
+amp-truncate-text[i-amphtml-truncate-expanded] {
+  overflow: visible !important;
```

Comment:
Is this enough? We should also be updating `height` and also likely we'll have issues with container being configured with `position:absolute`. Some of the JSS styles would need to be here as well.

Reply:
Agree on all above; some of this CSS is likely why I'm having trouble with the AMP component sizing/laying out correctly. You can expect this to change either in this PR or a follow-up